src/routes/common/logging/logging.interface.ts

```diff
@@ -0,0 +1,8 @@
+export const LoggingService = Symbol('ILoggingService');
+
+export interface ILoggingService {
+  log(message: string, ...optionalParams: any[]): void;
+  debug(message: string, ...optionalParams: any[]): void;
+  error(message: string, ...optionalParams: any[]): void;
+  warn(message: string, ...optionalParams: any[]): void;
```

Comment:
Nit: it's safer to use `unknown` than `any`:

> Anything is assignable to `unknown`, but `unknown` isn't assignable to anything but itself and `any` without a type assertion or a control flow based narrowing. Likewise, no operations are permitted on an `unknown` without first asserting or narrowing to a more specific type.

```suggestion
  log(message: string, ...optionalParams: unknown[]): void;
  debug(message: string, ...optionalParams: unknown[]): void;
  error(message: string, ...optionalParams: unknown[]): void;
  warn(message: string, ...optionalParams: unknown[]): void;
```